### PR TITLE
FUSETOOLS-2697: Execute Fuse UI smoke tests after each PR

### PR DESCRIPTION
**Pull request by apodhrad**
Signed-off-by: Andrej Podhradsky <apodhrad@redhat.com>

### CODE REVIEW

File: uitests/README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@
 
 ## Executing tests from command line
 
-The UI tests are disabled by default. To enable them you need to set
+The UI tests are enbled by default. To disable them you need to set
```
Comment (apupier) [inline uitests/README.md]: typo enbled -> enabled

**Comment by apupier** [comment]
What do you think to rebase on master before merging into master (as I was superslow to review it)?

**Comment by apodhrad** [comment]
rebased

**Comment by apodhrad** [comment]
this PR can be merged once we have a working job on dev jenkins

**Comment by apodhrad** [comment]
testPR

**Comment by apodhrad** [comment]
On fusesource the UI tests passed. Only projecttemplates are failing.

**Comment by apodhrad** [comment]
testPR

**Comment by apodhrad** [comment]
testPR

**Comment by apodhrad** [comment]
@tsedmik all UI smoke tests passed

**Comment by apodhrad** [comment]
testPR

**Pull request merged**